app/src/checkupdatesdialog.cpp

```diff
@@ -1,4 +1,4 @@
-#include "checkupdatesdialog.h"
+﻿#include "checkupdatesdialog.h"
```

Comment:
Is this intentional? Back in #683, you yourself said no BOM was preferable for better cross-platform compatibility. Seen this on a bunch of your commits lately.

Reply:
Damn it, must be my editor. I will remove the BOM